helm/efk-stack-app/Chart.yaml

```diff
@@ -1,19 +1,10 @@
-# Copyright 2019 Viasat, Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License").
-# You may not use this file except in compliance with the License.
-# A copy of the License is located at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# or in the "license" file accompanying this file. This file is distributed
-# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
-# express or implied. See the License for the specific language governing
-# permissions and limitations under the License.
-
+annotations:
+  application.giantswarm.io/metadata: http://localhost/efk-stack-app-0.3.5-2d81bd191a40b9a3bf48936ab34ad65315a148ce.tgz-meta/main.yaml
+  application.giantswarm.io/readme: http://localhost/efk-stack-app-0.3.5-2d81bd191a40b9a3bf48936ab34ad65315a148ce.tgz-meta/README.md
+  application.giantswarm.io/values-schema: http://localhost/efk-stack-app-0.3.5-2d81bd191a40b9a3bf48936ab34ad65315a148ce.tgz-meta/values.schema.json
```

Comment:
Whats up with these links to `localhost`?

Reply:
Someone hasn't changed catalog URL in the config ;)